drivers/gpu/drm/vc4/vc4_vec.c

```diff
@@ -370,7 +370,9 @@ static const struct drm_prop_enum_list legacy_tv_mode_names[] = {
 enum drm_connector_tv_mode
 vc4_vec_get_default_mode(struct drm_connector *connector)
 {
-	if (vc4_vec_tv_norm) {
+	if (connector->cmdline_mode.tv_mode_specified) {
+		return connector->cmdline_mode.tv_mode;
+	} else if (vc4_vec_tv_norm) {
```

Comment:
Are the enums still mismatched (e.g. NTSC-443, NTSC-J order)?

Reply:
The vc4 driver internally only uses the `DRM_MODE_TV_MODE_` enums.

The legacy `VC4_VEC_TV_MODE_` enums are only used if userspace uses the legacy (and non-standardised) "mode" property, and it converts to/from `DRM_MODE_TV_MODE_` on the fly.

The tv_norm module parameter is matching on strings, and is now using `drm_get_tv_mode_from_name` to match against the standardised enum strings and provide a `DRM_MODE_TV_MODE_`